### DISCUSSION TITLE
project remake

### DISCUSSION THREAD

**Discussion by AAnemeCC** [raspberrypi/picamera2#575] [Q&A]
Hello, I would like to ask if it would be possible and how to remake this code to work with picamera2.
https://github.com/gigafide/pi-smart-car/blob/master/object_detection/car_detector.py

**Comment by davidplowman** [comment]
Yes, maybe start from an example like [this](https://github.com/raspberrypi/picamera2/blob/main/examples/opencv_face_detect.py).  You need to do is get the image array, and then replace the OpenCV image analysis.
